src/header/css/header-menus.pcss

```diff
@@ -402,17 +483,31 @@
 		:matches([data-filled="13"], [data-filled="14"]) &:nth-child(n+8),
 		:matches([data-filled="15"], [data-filled="16"]) &:nth-child(n+9),
 		:matches([data-filled="17"], [data-filled="18"]) &:nth-child(n+10) {
-			margin-inline-start: calc(50% + 10px);
+			margin-inline-start: calc(33% + 10px);
 		}
 
 		@nest [data-filled]:not([data-filled=""]) & {
-			width: calc(50% - 10px);
+			width: calc(100% - 10px);
 		}
 
 		@nest [data-filled]:not([data-filled=""]) &:last-child:nth-child(odd) {
 			margin-block-end: 60px;
 		}
 	}
+
+	@media (--width-l) {
+		&.heading {
```

Comment:
this class is too generic, should use prefixes, like `.esri-header-menus-subitem--heading`. Then you can avoid nesting. The reason for this is that having a class called `heading` has a high probability for class collisions as many sites will have a class called `heading`

Reply:
@paulcpederson this has been updated..